### DISCUSSION TITLE
Upgrading Version 9.6 version available

### DISCUSSION THREAD

**Discussion by naddanki87** [rohitbemax/dataimporthandler#82] [Q&A]
I am reaching out to discuss my recent pull request (https://github.com/SearchScale/dataimporthandler/pull/81) regarding the compatibility of Solr version 9.6.1 with the package version constraint of 9.4.0.

The current version constraint poses issues for users looking to upgrade to Solr 9.6.1, as it leads to compatibility errors. My proposed changes aim to resolve this problem by updating the version constraints to accommodate 9.6.1, ensuring smoother upgrades for the community.

I would greatly appreciate your review and consideration of this pull request. If there are any concerns or required adjustments, please let me know. I'm eager to contribute to the project and support the community.

**Comment by aumareddy** [comment]
HI @naddanki87 ,

I have requirement .to import data from snowflake to Apache solr.  can you please provide to how to do it any steps please provide.
